plugins/scheduler-docker-local/check-deploy

```diff
@@ -99,20 +98,13 @@ scheduler-docker-local-check-deploy() {
   }
   trap "checks_check_deploy_cleanup $DOKKU_APP_CONTAINER_ID" RETURN INT TERM EXIT
 
+  # We allow custom check for web instances only
   if [[ ! -s "${CHECKS_FILENAME}" ]] || [[ "$DOKKU_APP_CONTAINER_TYPE" != "web" ]]; then
-    # We allow custom check for web instances only
-    if [[ "$DOKKU_APP_CONTAINER_TYPE" == "web" ]]; then
-      dokku_log_verbose "CHECKS file not found in container: Running simple container check..."
-      dokku_log_verbose "For more efficient zero downtime deployments, create a file CHECKS. See ${CHECKS_URL} for examples"
-    else
-      dokku_log_verbose "Non web container detected: Running default container check..."
-    fi
-
     rm -rf "$CHECK_DEPLOY_TMP_WORK_DIR" &>/dev/null || true
 
     # simple default check to see if the container stuck around
     local DOKKU_DEFAULT_CHECKS_WAIT="${DOKKU_DEFAULT_CHECKS_WAIT:-10}"
-    dokku_log_info1 "Waiting for $DOKKU_DEFAULT_CHECKS_WAIT seconds ..."
+    dokku_log_verbose "Waiting for $DOKKU_DEFAULT_CHECKS_WAIT seconds ..."
```

Comment:
i think this particular line should stay info1 in order to communicate to the user that we're intentionally not proceeding yet